docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "How-To: Load Data in ML.NET"
+description: Load data file and streaming data into ML.NET
+ms.date: 04/03/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to load data from file and streaming data sources.
+---
+
+# How-To: Load Data in ML.NET
+
+This how-to shows you how to load data for processing and training into ML.NET that's stored in files or real-time / streaming data sources.
+
+## Create Data Model Schema
+
+ML.NET allows for defining data model schemas via classes. For example, given the following input data:
+
+```text
+Size (Sq. ft.), HistoricalPrice1 ($), HistoricalPrice2 ($), HistoricalPrice3 ($), Current Price ($)
+700, 100000, 3000000, 250000, 500000
+1000, 600000, 400000, 650000, 700000
+```
+
+Create a data model that looks like the snippet below:
+
+```csharp
+class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+    [LoadColumn(1,3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+    [LoadColumn(4)]
+    public float CurrentPrice { get; set; }
+}
+```
+
+### Column Attributes
+
+ML.NET provides additional ways to extend data models through attributes.
+
+The [`LoadColumn`](xref:Microsoft.ML.Data.LoadColumnAttribute) attribute specifies your properties' column indices.
+
+Load columns as: 
+- Individual columns like `Size` and `CurrentPrices` in the previous `HousingData` class example.
+- Multiple columns at a time in the form of a vector like `HistoricalPrices` in the previous `HousingData` class example.
+
+If you have a vector property, use the [`VectorType`](xref:Microsoft.ML.Data.VectorTypeAttribute) attribute in your model. One thing to note however is that all of the elements in the vector need to be of the same type.
+
+## Load Data From A File
+
+To load data from a file use the [`LoadFromTextFile`](xref:Microsoft.ML.TextLoaderSaverCatalog.LoadFromTextFile*) method along with the data model schema for the data to be loaded. Since `separatorChar` parameter is tab-delimited by default, change it for your data file as needed. If your file has a header, set the `hasHeader` parameter to `true`.
+
+```csharp
+//Create MLContext
+MLContext mlContext = new MLContext();
+
+//Load Data
+IDataView data = mlContext.Data.LoadFromTextFile<HousingData>("my-data-file.csv", separatorChar: ',', hasHeader: true);
```

Comment:
Do you think it's worth loading it back into an IEnumerable and printing it out? And mentioning that the IDataView is lazy?

Reply:
Mentioning lazy loading would be best in the ML.NET concepts article and the printing out / inspection I can put in the article on inspecting values (loading, transforms, predictions).